consensus/core/build.rs

```diff
@@ -146,6 +155,15 @@ fn build_anemo_services(out_dir: &Path) {
                 .codec_path(codec_path)
                 .build(),
         )
+        .method(
+            anemo_build::manual::Method::builder()
```

Comment:
Since anemo is not used in prod, we can do incompatible changes as well.

Reply:
Good call, updated!